internal/cluster.go

```diff
@@ -181,9 +182,22 @@ func (clusterService *ClusterService) initMembershipListener(connection *Connect
 	}
 	registrationId := ClientAddMembershipListenerDecodeResponse(response)()
 	wg.Wait() //Wait until the inital member list is fetched.
+	clusterService.logMembers()
 	log.Println("Registered membership listener with Id ", *registrationId)
 	return nil
 }
+
+func (clusterService *ClusterService) logMembers() {
+	members := clusterService.members.Load().([]*Member)
+	membersInfo := fmt.Sprintf("New member list:\n\nMembers[%d]{\n", len(members))
+	for _, member := range members {
+		memberInfo := fmt.Sprintf("\t Member [%s:%d]\n", member.Address().Host(), member.Address().Port())
```

Comment:
A member string should be same as : https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/instance/AbstractMember.java#L181